packages/devtools/lib/src/timeline/frame_flame_chart.dart

```diff
@@ -349,6 +355,9 @@ class FlameChartItem {
       _labelWrapper.style.maxWidth = '${_startingWidth}px';
     }
     style.top = '${_top}px';
+
+    // TODO(kenzie): make flame chart item appear selected.
```

Comment:
Perhaps also change the cursor for `.flame-chart-item` to something like `pointer`?

Reply:
done.